### PR TITLE
Add the waiting page for file upload

### PR DESCRIPTION
**Pull request by PXke**

### CODE REVIEW

File: modules/websubmit/lib/websubmit_templates.py [hunk 1]
```diff
@@ -797,16 +797,18 @@ def tmpl_page_interface_js(self, ln, upload, field, fieldhtml, txt, check, level
...
-                           }
-                         }"""
+                out += """if (tester2()) {
+                            $(this).attr("disabled", true);
+                            document.forms[0].action="/submit";
+                            document.forms[0].step.value=1;
+                            user_must_confirm_before_leaving_page = false;
+                            document.forms[0].submit();
+                            $("form").find("button:submit").parent().hide();
+                 $("form").find("button:submit").parent().parent().append("<div id='waitarea'><h2>Submission in progress, please be patient!</h2> <b>You will be redirected automatically</b><img src='https://ir.tind.io/img/loading.gif'> </div>");
```
Comment (frecar) [inline modules/websubmit/lib/websubmit_templates.py]: I guess we should not rely on ir.tind.io for serving the spinner?

Comment (PXke) [inline modules/websubmit/lib/websubmit_templates.py]: Indeed !

**Comment by frecar** [comment]
retest now please

**Comment by frecar** [comment]
retest now

**Comment by frecar** [comment]
retest now

File: modules/websubmit/lib/websubmit_templates.py (hunk 1 above)
Comment (frecar) [inline modules/websubmit/lib/websubmit_templates.py]: Change it to something generic, like the CFG_SITE_URL ? Then we merge it

**Pull request merged**